README.md

```diff
@@ -1,6 +1,227 @@
+
+
+## Documentation
+[Documentation](/app-services-sdk-python/docs)
+## Examples
+[Examples](/app-services-sdk-python/examples)
+
+# RHOAS SDK for Java
+
+Java packages and API clients for RHOAS services
+
+## Prequisites
+
+- [NodeJS 14.x lts](https://nodejs.org/en/about/releases/) or above
+- [Maven 3.6.x](https://maven.apache.org) or above
+- [YQ 4.6.x](https://github.com/mikefarah/yq#install) or above
+
+Typically developers need to integrate with Management SDK's to create or manage service instances
+and later use upstream libraries or management SDK's to interact with services.
+
+You can view all SDK packages directly in [GitHub](https://github.com/orgs/redhat-developer/packages)
+
+## Management SDK's
+
+| Name             | Artifact Id                        | Status | Version           | Package            | Docs               |
+| ---------------- | :--------------------------------- | :----- | ----------------- | ------------------ | ------------------ |
+| kafka            | **kafka-management-sdk**           | beta   | 1.1.2             | [info](https://search.maven.org/search?q=a:kafka-management-sdk)                | [doc][kafkagit]    |
+| service accounts            | **service-accounts-sdk**           | alpha   | 1.1.2             | [info](https://search.maven.org/search?q=a:service-accounts-sdk)                | [doc][sadoc]    |
+| service registry | **registry-management-sdk**        | alpha  | 0.0.6             | [info](https://search.maven.org/search?q=a:registry-management-sdk)                | [doc][smarteventsgit] |
+|  connector-management | **connector-management-sdk**        | alpha  | 0.0.6             | [info](https://search.maven.org/search?q=a:connector-management-sdk)                | [doc][connectorgit] |
+|  smart-events-management | **smart-events-management-sdk**        | alpha  | 0.0.6             | [info](https://search.maven.org/search?q=a:smartevents-management-sdk)                | [doc][smarteventsgit] |
+
+## Instances SDK's
+
+| Name             | Package name                  | Status | Version            |  Package            | Docs                |
+| ---------------- | :---------------------------- | :----- | ------------------ | ------------------- | ------------------- |
+| kafka            | **kafka-instance-sdk**        | beta   | 1.1.2              | [info](https://search.maven.org/search?q=a:kafka-instance-sdk)                 | [doc][kinstancegit] |
+| service registry | TBA                           | TBA    | TBA                | TBA                 | TBA                 |
+
+## Docs
+
+[Documentation](/app-services-sdk-java/docs)
+
+# RHOAS SDK for Go
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/redhat-developer/app-services-sdk-go.svg)](https://pkg.go.dev/github.com/redhat-developer/app-services-sdk-go)
+
+Go packages and API clients for Red Hat OpenShift Application Services (RHOAS) 
+
+## Prequisites
+
+- [Go 1.15](https://golang.org/doc/go1.15) or above
```

Comment:
`1.15` ?

Reply:
The go SDK in core uses `go 1.15` in the mod file so to use it you need to at least be able to compile go `1.15` with what ever version of the compiler you have. Is this wrong maybe I am misunderstanding how it works

Reply:
fair enough 👍